Adyen/Service/Checkout.cs

```diff
@@ -184,5 +184,17 @@ public PaymentLinkResource PaymentLinks(CreatePaymentLinkRequest createPaymentLi
             var jsonResponse = _paymentLinksResult.Request(jsonRequest);
             return JsonConvert.DeserializeObject<PaymentLinkResource>(jsonResponse);
         }
+
+        /// <summary>
+        /// POST /paymentsLinks API call async
+        /// </summary>
+        /// <param name="createPaymentLinkRequest"></param>
+        /// <returns>CreatePaymentLinkResponse</returns>
```

Comment:
minor
```suggestion
        /// <returns>PaymentLinkResource</returns>
```
not sure if `Task` should be included too